src/screens/FeedScreen/Header.js

```diff
@@ -206,7 +206,7 @@ const _renderProfileNormal = ({
             <View style={{}}>
               <Image
                 source={{
-                  uri: profile_pic_url ?? DEFAULT_PROFILE_PIC_PATH
+                  uri: profile_pic_url || DEFAULT_PROFILE_PIC_PATH
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from the nullish coalescing operator (`??`) to the logical OR operator (`||`) could potentially introduce bugs. The `??` operator only defaults to `DEFAULT_PROFILE_PIC_PATH` if `profile_pic_url` is `null` or `undefined`. However, the `||` operator will also default to `DEFAULT_PROFILE_PIC_PATH` if `profile_pic_url` is any falsy value, such as an empty string (`""`). If an empty string is a valid value for `profile_pic_url`, this change could lead to unexpected behavior.
```diff
- uri: profile_pic_url || DEFAULT_PROFILE_PIC_PATH
+ uri: profile_pic_url ?? DEFAULT_PROFILE_PIC_PATH
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->